socorro/collector/crashmover_app.py

```diff
@@ -26,67 +26,6 @@ class CrashMoverApp(FetchTransformSaveApp):
 
 
 #==============================================================================
-class ProcessedCrashCopierApp(FetchTransformSaveWithSeparateNewCrashSourceApp):
```

Comment:
This app seems like it's used to move processed crash data from one crash storage to another. It'd be used if we migrated to a new data store or maybe to a new ES cluster or something like that. We haven't done that in a while and I don't think we're likely to use this to do that sort of thing in the future.

There are no tests for this and it defaults to the TarFileCrashStore. Given that, I nixed it.